concent_api/core/models.py

```diff
@@ -518,3 +519,67 @@ def clean(self):
             raise ValidationError({
                 'pending_response': 'PaymentInfo should be related with Pending Response'
             })
+
+
+class GlobalTransactionState(Model):
+    """
+    Represents state of whole transaction service by storing last `nonce` used.
+
+    There should always be at most one object of this type with id = 0. If it does not exist, it should be created and
+    nonce initialized with a value obtained from SCI (Client.get_transaction_count()).
+    """
+
+    nonce = DecimalField(max_digits=32, decimal_places=0, default=0, unique=True)
+
+    def save(self, *args, **kwargs):  # pylint: disable=arguments-differ
+        self.pk = 0
+        super().save(*args, **kwargs)
+
+    @property
+    def get_nonce(self):
+        return int(self.nonce)
+
+
+class PendingEthereumTransaction(Model):
+    """
+    Represents pending Ethereum transaction state.
+    """
+
+    nonce = DecimalField(max_digits=78, decimal_places=0)
+
+    gasprice = DecimalField(max_digits=78, decimal_places=0)
+    startgas = DecimalField(max_digits=78, decimal_places=0)
+    value = DecimalField(max_digits=78, decimal_places=0)
+
+    to = BinaryField(max_length=20)
+    data = BinaryField()
+
+    v = IntegerField()
+    r = DecimalField(max_digits=78, decimal_places=0)
+    s = DecimalField(max_digits=78, decimal_places=0)
+
+    created_at = DateTimeField(auto_now_add=True)
+
+    @property
+    def get_nonce(self):
+        return int(self.nonce)
```

Comment:
This sounds like a function name. Properties should be named like variables (i.e. using nouns, not verbs). E.g. `nonce_as_int`.

Reply:
Deleted, just as you said in next comment